packages/p2p/src/components/verification/verification.tsx

```diff
@@ -24,10 +26,10 @@ const VerificationWrapper = ({ should_wrap, children }) => {
         );
     }
 
-    return children;
+    return <>{children}</>;
 };
 
-const Verification = ({ should_wrap }) => {
+const Verification = ({ should_wrap = false }: TVerificationProps) => {
```

Comment:
please remove `should_wrap` since it's not being used then we can also remove `VerificationWrapper `